awscli/examples/medical-imaging/search-image-sets.rst

```diff
@@ -0,0 +1,120 @@
+**Example 1: To search image sets with an EQUAL operator**
+
+The following ``search-image-sets`` code example uses the EQUAL operator to search image sets based on a specific value. ::
+
+    aws medical-imaging search-image-sets \
+        --datastore-id 12345678901234567890123456789012 \
+        --search-criteria '{
```

Comment:
Needs line continuation marks for lines 7-11.

Reply:
Continuation marks are not correct because of the single quote. This is within a string and line continuation marks will create an error.

Instead I am following the practice of using a json file, which is used elsewhere in the cli examples. I will do this for all of these quoted json strings.

```
  aws medical-imaging search-image-sets \
        --datastore-id 12345678901234567890123456789012 \
        --search-criteria file://search-criteria.json

Contents of ``search-criteria.json`` ::

    {
        "filters": [{
            "values": [{"DICOMPatientId" : "SUBJECT08701"}],
            "operator": "EQUAL"
        }]
    }

```